### PR TITLE
Azerbaijani Language

### PR DESCRIPTION
**Pull request by kongulov**
1) Translate fix
2) Currency Test
3) Add Noun Gender Inflector
4) Add Noun Gender Inflector Test

### CODE REVIEW

**Comment by kongulov** [comment]
Helo @kwn 
Can you please check and merge?

File: src/CurrencyTransformer/AzerbaijaniCurrencyTransformer.php [hunk 1]
```diff
@@ -37,7 +37,7 @@ public function toWords(int $amount, string $currency, ?CurrencyTransformerOptio
 
         if (!array_key_exists($currency, AzerbaijaniDictionary::$currencyNames)) {
             throw new NumberToWordsException(
-                sprintf('Currency "%s" is not available for "%s" language', $currency, get_class($this))
+                sprintf('"%s" valyutası "%s" dilində mövcud deyil', $currency, get_class($this))
```
Comment (kwn) [inline src/CurrencyTransformer/AzerbaijaniCurrencyTransformer.php:40]: The exceptions and generic messages should be all in English. Could you please revert it?

File: tests/Language/Azerbaijani/AzerbaijaniNounGenderInflectorTest.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+class AzerbaijaniNounGenderInflectorTest extends TestCase
+{
+    private static array $nouns = ['fincan', 'fincanlar', 'fincanlar'];
+
+    /**
+     * @dataProvider providerItInflectsNounsByNumbers
+     */
+    public function testItInflectsNounsByNumbers($number, $expectedNoun): void
+    {
+        $polishInflector = new AzerbaijaniNounGenderInflector();
```
Comment (kwn) [inline tests/Language/Azerbaijani/AzerbaijaniNounGenderInflectorTest.php:16]:
```suggestion
        $azerbaijaniInflector = new AzerbaijaniNounGenderInflector();
```

File: tests/Language/Azerbaijani/AzerbaijaniNounGenderInflectorTest.php [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+        $inflected = $polishInflector->inflectNounByNumber($number, self::$nouns[0], self::$nouns[1], self::$nouns[2]);
```
Comment (kwn) [inline tests/Language/Azerbaijani/AzerbaijaniNounGenderInflectorTest.php:18]:
```suggestion
        $inflected = $azerbaijaniInflector->inflectNounByNumber($number, self::$nouns[0], self::$nouns[1], self::$nouns[2]);
```

**Review by kwn — CHANGES_REQUESTED** [review]
Thanks. Just a few minor comments added.

**Comment by kongulov** [comment]
@kwn Thanks. All is ready

**Comment by kwn** [comment]
Thanks @kongulov. Released in `2.4.3`.

**Pull request merged**